src/Appwrite/Platform/Tasks/ScheduleExecutions.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Appwrite\Event\Func;
+use Utopia\Database\Database;
+use Utopia\Pools\Group;
+
+class ScheduleExecutions extends ScheduleBase
+{
+    public const UPDATE_TIMER = 3; // seconds
+    public const ENQUEUE_TIMER = 4; // seconds
+
+    public static function getName(): string
+    {
+        return 'schedule-executions';
+    }
+
+    public static function getSupportedResource(): string
+    {
+        return 'execution';
+    }
+
+    protected function enqueueResources(Group $pools, Database $dbForConsole): void
+    {
+        $queue = $pools->get('queue')->pop();
+        $connection = $queue->getResource();
+        $queueForFunctions = new Func($connection);
+
+        foreach ($this->schedules as $schedule) {
+            if (!$schedule['active']) {
+                unset($this->schedules[$schedule['resourceId']]);
+                continue;
+            }
+
+            $now = new \DateTime();
+            $scheduledAt = new \DateTime($schedule['schedule']);
+
+            if ($scheduledAt > $now) {
+                continue;
+            }
+
+            $queueForFunctions
+                ->setType('schedule')
+                ->setFunctionId($schedule['resource']['functionId'])
+                ->setExecution($schedule['resource'])
+                ->setProject($schedule['project'])
+                ->trigger();
```

Comment:
As mention in API, here we need to also set stuff like body, headers, method, path, user jwt...
To get those data here, we could add attribute to schedules as temporary storage for those info (since we delete the document right after)

Reply:
Went with schedule for now